architecture/backend/database-sceme/README.md

```diff
@@ -0,0 +1 @@
+# team4_docs
```

Comment:
Please, add a short description of the database model. ER diagram is a bit overcomplicated as it shows the big picture of the DB. There we can find users, roles, language, questions storage, tests, etc. A short and splitted description of the data could make everything much simpler.